scalableminds/webknossos
# Annotation flag to enum

### Changes:
 - Previously the state of an annotaion was defined by the combination of the flags "isAssigned", "isFinished" and "isInProgress".
- Now we use an Enum for this with the following Values: "Unassigned", "Assigned", "InProgress", "Finished".
- The state in the DB is represented as a string ( "Unassigned", "Assigned", "InProgress", "Finished")
- Overview: [SCMstate.txt](https://github.com/scalableminds/webknossos/files/1597652/SCMstate.txt)



### Steps to test:
- try the route '/api/user/annotations'
- try the route '/api/users/$id<[^/]+>/annotations'
and check if you get the expected result.

### Issues:
- fixes #2187 

------
